### PR TITLE
create new django app for wikimedia

### PR DESCRIPTION
**Pull request by maneeshatejani1**
Added new django app for client features
dummy view has been added for testing. Will be removed once development starts

### CODE REVIEW

File: openedx/features/wikimedia_features/tests.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```
Comment (tehreem-sadat) [inline openedx/features/wikimedia_features/tests.py]: Can we please remove this extra file ? We can create this file later on when needed.

File: openedx/features/wikimedia_features/apps.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
```
Comment (tehreem-sadat) [inline openedx/features/wikimedia_features/apps.py]: wikimedia_features will be a container, in which we'll have dashboard app.

File: openedx/features/wikimedia_features/views.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from django.shortcuts import render
+
+# Create your views here.
+
+import json
+from django.http import HttpResponse
+
+def dummy(request):
```
Comment (tehreem-sadat) [inline openedx/features/wikimedia_features/views.py]: Can we please remove these dummy views ?

**Review by tehreem-sadat — CHANGES_REQUESTED** [review]

File: openedx/features/wikimedia_features/tests.py (hunk 1 above)
Comment (maneeshatejani1) [reply openedx/features/wikimedia_features/tests.py]: sure

File: openedx/features/wikimedia_features/apps.py (hunk 2 above)
Comment (maneeshatejani1) [reply openedx/features/wikimedia_features/apps.py]: sure

File: openedx/features/wikimedia_features/views.py (hunk 3 above)
Comment (maneeshatejani1) [reply openedx/features/wikimedia_features/views.py]: sure

File: lms/envs/common.py [hunk 4]
```diff
@@ -3090,6 +3090,9 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     # in the LMS process at the moment, so anything that has Django admin access
     # permissions needs to be listed as an LMS app or the script will fail.
     'user_tasks',
+
+    'openedx.features.wikimedia_features.reporting_dashboard',
```
Comment (tehreem-sadat) [inline lms/envs/common.py]:
```suggestion
    'openedx.features.wikimedia_features.dashboard',
```

**Review by tehreem-sadat — COMMENTED** [review]
Just one minor suggestion.

**Review by tehreem-sadat — APPROVED** [review]

**Pull request merged**